client/tos/snackbar/style.scss

```diff
@@ -0,0 +1,5 @@
+.woocommerce-payments__tos-snackbar {
+	position: fixed;
+	bottom: 0;
+	margin: 0 0 20px 20px;
+}
```

Comment:
One more reason to use wc-admin notices if possible :)

Reply:
It is used now :)